exhibitor-appliance.yaml

```diff
@@ -19,6 +19,12 @@ SenzaInfo:
     - CustomS3BucketRegion:
         Description: "Use custom s3 bucket region for exhibitor configuration"
         Default: ""
+    - CustomTransactionsDirectory:
+        Description: "Use custom transactions directory"
+        Default: "/opt/exhibitor/transactions"
```

Comment:
If directory already exists it wont change permissions.

Best approach...
Actually It looks fine, just do a small change `/opt/exhibitor/ => /opt/zookeeper/` to be sure that nothing will break with updated senza template.